scripts/benchmarks.py

```diff
@@ -109,6 +109,17 @@ def accept(seed, testdata, value):
     return accept
 
 
+def lower_bound(seed, testdata, value):
+    rnd = random.Random(seed)
+    for b in testdata.buffer:
+        c = rnd.randint(0, 255)
+        if c < b:
+            return True
+        if c > b:
+            return False
+    return True
```

Comment:
Alright, let's put that down to reviewing the diff instead if the file and stop here :smile: